src/Symfony/Component/DependencyInjection/ServiceProviderInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection;
+
+use Psr\Container\ContainerInterface as PsrContainerInterface;
+
+/**
+ * A ServiceProviderInterface exposes provided service types via the {@link getProvidedServices} method.
+ */
```

Comment:
Not only types but identifiers, I would change it to something like `Exposes the services provided by a container.` to be consistent with PSR-11. `@author` annotations are missing also

Reply:
does it sound better now?

Reply:
I would drop the `via the {@link getProvidedServices} method.` part (no need for describing the api, the code speaks for itself), but good enough :)